ibm_catalog.json

```diff
@@ -38,7 +38,7 @@
         },
         {
           "title": "IBM Cloud service integrations",
-          "description": "Seamless integration with [Container Registry](https://www.ibm.com/products/container-registry), Activity Tracker, [Cloud Logs](https://www.ibm.com/products/cloud-logs) and [Cloud Monitoring](https://www.ibm.com/products/cloud-monitoring)."
+          "description": "Seamless integration with [Container Registry](https://www.ibm.com/products/container-registry), Activity Tracker, [Cloud Logs](https://www.ibm.com/products/cloud-logs), [Cloud Monitoring](https://www.ibm.com/products/cloud-monitoring), SCC Workload Protection](https://www.ibm.com/products/security-and-compliance-center), and App Configuration."
```

Comment:
Why no links to Activity Tracker and App Configuration docs? Also your missing an opening square bracket for SCC WP

Reply:
App Configuration is not a direct dependency, it should not be mentioned here. Its only used as a config aggregator for SCC-WP - its not used for feature flagging

Reply:
done